### PR TITLE
[IA-2919] Label runtimes with workspace namespace value

### PR DESCRIPTION
**Pull request by kyuksel**
This is the first PR for the ticket [IA-2919](https://broadworkbench.atlassian.net/browse/IA-2919). There will be another PR for the remainder of the work once the [requisite backend work](https://broadworkbench.atlassian.net/browse/IA-2958) is in place. I wanted to get this PR's changes in so we start capturing `workspace namespace` (aka `billing project`) values for runtimes before the One Project Per Workspace (PPW) work is released to `prod` since `workspace namespace` may no longer be the same as `google project` after that release, which would cause Cloud Environments page to display wrong information in `Billing project` column.

Tested against `dev` using a local UI deploy.

### CODE REVIEW

File: src/components/data/data-utils.js [hunk 1]
```diff
@@ -245,12 +245,7 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
       if (useFireCloudDataModel) {
         await workspace.importEntitiesFile(file)
       } else {
-        let filesize = Number.MAX_SAFE_INTEGER
-        try {
-          filesize = file.size
-        } catch (err) {
-          // noop
-        }
```
Comment (kyuksel) [inline src/components/data/data-utils.js:253]: Refactoring. Not ticket related. Wanted to simplify the code.

File: src/components/data/data-utils.js [hunk 2]
```diff
@@ -259,7 +254,6 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
           notifyDataImportProgress(jobId)
         }
       }
-
```
Comment (kyuksel) [inline src/components/data/data-utils.js:262]: Wanted to make the method newlining consistent.

File: src/pages/Environments.js [hunk 3]
```diff
@@ -466,20 +466,20 @@ const Environments = () => {
             }
           },
           {
```
Comment (kyuksel) [inline src/pages/Environments.js:468]: Not directly related to the ticket. 
The diff below can be misleading. There is no logic change. I merely swapped the positions of the `Status` and `Location` columns for `Your Disks` table so it's consistent with `Your Cloud Environments` table. However, git doesn't seem to have realized that.

**Old:**
image

**New:**
image

File: src/pages/workspaces/workspace/Data.js [hunk 4]
```diff
@@ -450,7 +450,7 @@ const WorkspaceData = _.flow(
   const getRunningImportJobs = async () => {
     try {
       const runningJobs = await Ajax(signal).Workspaces.workspace(namespace, name).listImportJobs(true)
-      _.map(job => {
```
Comment (kyuksel) [inline src/pages/workspaces/workspace/Data.js:453]: Not ticket related. Wanted to get rid of the linting error to reduce noise.

File: src/components/data/data-utils.js [hunk 5]
```diff
@@ -245,12 +245,7 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
       if (useFireCloudDataModel) {
         await workspace.importEntitiesFile(file)
       } else {
-        let filesize = Number.MAX_SAFE_INTEGER
-        try {
-          filesize = file.size
-        } catch (err) {
-          // noop
-        }
+        const filesize = file?.size || Number.MAX_SAFE_INTEGER
```
Comment (petesantos) [inline src/components/data/data-utils.js:248]: nice opportunistic refactor

**Review by petesantos — APPROVED** [review]
LGTM

File: src/pages/Environments.js (hunk 3 above)
Comment (rtitle) [reply src/pages/Environments.js:468]: Nice 👍

**Review by rtitle — APPROVED** [review]
Looks good, I did a quick test to create a runtime+disk and verified the label existed.

Do apps already have the `saturnWorkspaceNamespace` label?

**Review by Qi77Qi — APPROVED** [review]
lgtm

**Comment by kyuksel** [comment]
@rtitle @Qi77Qi Thanks for the review and testing.

> Do apps already have the saturnWorkspaceNamespace label?

yes

**Pull request merged**